app/views/layouts/application.html.haml

```diff
@@ -57,3 +57,15 @@
 
     = include_chartbeat
     = include_mixpanel_guid
+
+    #blueimp-gallery.blueimp-gallery.blueimp-gallery-controls
```

Comment:
Classes should be listed before IDs (.blueimp-gallery should precede #blueimp-gallery)